platforms/dir.proj

```diff
@@ -17,6 +17,9 @@
     <Project Include="xamarin.mac/*.csproj">
       <TargetGroup>xamarin.mac</TargetGroup>
     </Project>
+    <Project Include="xamarin.mac/Facades/*.csproj">
```

Comment:
I'm fine with keeping them in a sub-dir if that makes syncing them easier but in the end they aren't treated any different then the others here.